server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
Let's not change the existing stuff that is proven to work, also I'd rather not drop DEFAULT value for data creation column just for the sake of ORM :-}